model/res/src/main/scala/aqua/res/ResolvedOp.scala

```diff
@@ -50,6 +50,10 @@ case class CallServiceRes(
   override def toString: String = s"(call $peerId ($serviceId $funcName) $call)"
 }
 
+case class ApStreamMapRes(key: LiteralModel, value: ValueModel, exportTo: CallModel.Export) extends ResolvedOp {
```

Comment:
Are only literals supported as keys?

Reply:
for now - yes

Reply:
It is AquaVM limitation or ours? I mean, I do not see any problems in code for allowing `ValueModel` here